morfeus/buried_volume.py

```diff
@@ -393,6 +402,7 @@ def compute_distal_volume(
                 center=self._sphere.center,
                 radius=new_radius,
                 density=self._sphere.density,
+                method=self._method,
```

Comment:
I think it's more consistent to use the same method as when creating the `BuriedVolume` object?